### ISSUE TITLE
Add Pixelbook Support

### ISSUE THREAD

**Issue by scarolan** [Yubico/yubioath-android#112]
I'm able to install this Android app on my Pixelbook but it never recognizes the Yubikey.  You should fix this or remove the compatibility in the play store to prevent chromebook users from thinking it is supported.

**Comment by imakhalova** [comment]
Have you tried USB or NFC version? Do you have any logs?

**Comment by dainnilsson** [comment]
Closing this as a duplicate of #60. Unfortunately we're currently blocked on getting this to work due to limitations of Android on ChromeOS.
